sisyphus/job.py

```diff
@@ -289,17 +289,18 @@ def _sis_setup_directory(self, force=False):
             pickle.dump(self, f)
 
         with open(self._sis_path(gs.JOB_INFO), "w", encoding="utf-8") as f:
-            for tag in self.tags:
+            for tag in sorted(str(t) for t in self.tags):
                 f.write("TAG: %s\n" % tag)
-            for i in self._sis_inputs:
+            for i in sorted(str(p) for p in self._sis_inputs):
                 f.write("INPUT: %s\n" % i)
-            for key, value in self._sis_kwargs.items():
+            for key in sorted(self._sis_kwargs.keys()):
```

Comment:
I would prefer to keep the original order. The order is well defined here (insertion order).